src/main/java/com/treasure/hunt/game/mods/hideandseek/HideAndSeekGameEngine.java

```diff
@@ -22,7 +22,7 @@ public HideAndSeekGameEngine(Searcher searcher, Hider hider) {
      * and then gives his hint
      */
     protected void moveHider() {
-        treasurePos = hider.getTreasureLocation(); // Difference between GameEngine and HideAndSeekGameEngine.
+        treasurePos = hider.getTreasurePos(); // Difference between GameEngine and HideAndSeekGameEngine.
```

Comment:
I remember @axel1200 saying that abbreviations in method/variable/class-names are bad. What has changed?

Reply:
Well this was @anLeid and I have no idea why we do this @anLeid could u tell us.

Reply:
Why do we rename it at all